data_preprocess/transformer_tok.py

```diff
@@ -0,0 +1,26 @@
+from transformers import AutoTokenizer
+import torch
+import unidecode
+import re
+
+def read_file(filename):
+    """
+    Read the text file corresponding to radiology reports. 
+    Normalize the text. 
+
+    """
+    with open(filename, 'r', encoding='utf-8') as file:
+        text = file.read()
+    text = ' '.join(re.sub('[^A-Za-z ]+', '', text).lower().split())
+    return unidecode.unidecode(text)
```

Comment:
Let's move this function into a separate file & import it as needed, OR let's just import the one from `text_tokenize.py` into this file